samples/SmokeTest/CosmosDB.ts

```diff
@@ -0,0 +1,99 @@
+import {CosmosClient, Database, Container} from "@azure/cosmos";
+
+export class CosmosDB {  
+   
+    private static dataBaseName = "jsSolarSystem";
+    private static collectionName = "PlanetsCollection";
+    private static client : CosmosClient;
+    private static db : Database;
+    private static container : Container;
+
+    static async Run(){
+        console.log();
```

Comment:
You can use template literals to clean this up some!

```
console.log(`
------------------------
Cosmos DB
------------------------
1) Create a database
2) Create a collection
3) Create documents (items) in the collection
4) Delete the database (Clean up the resource)
`)
```

Reply:
Hi! I change those console.log blocks to use template literals, but the problem is that the code looses indentation in order to not print the tabs in the console:
```typescript
export class BlobStorage{

    private static ContainerClient : ContainerClient;
    private static blobName : string;

    static async Run(){
        console.log(`
------------------------
Storage - Blobs
------------------------
1) Upload Blob
2) Delete Blob (Clean up the resource
`);

        const account = process.env["STORAGE_ACCOUNT_NAME"]     || "<YourStorageAccountNAME>";
        const accountKey = process.env["STORAGE_ACCOUNT_KEY"]   || "<YourStorageAccountKEY>";
        const containerName = "mycontainer";
        BlobStorage.blobName = "JSNewBlob";
```

I found that one option could be doing something like this:
```typescript
    static async Run(){
        const message = (`
        ------------------------
        Storage - Blobs
        ------------------------
        1) Upload Blob
        2) Delete Blob (Clean up the resource)
        `).replace(/^        /gm, '');

        console.log(message);
```

What would you recommend? Do it without indentation, using the replace() method or going back to console.log() each line?